docs/java/how-to-develop-a-workflow-definition-in-java.md

```diff
@@ -0,0 +1,186 @@
+---
+id: how-to-develop-a-workflow-definition-in-java
+title: How to develop a Workflow Definition in Java
+sidebar_label: Workflow Definition
+description: In the Temporal Java SDK programming model, a Workflow is a class which implements a Workflow Interface.
+---
+
+import RelatedReadList from '../components/RelatedReadList.js'
+
+In the Temporal Java SDK programming model, a Workflow is a class which implements a Workflow Interface:
+
+```java
+public class FileProcessingWorkflowImpl implements FileProcessingWorkflow {
+  // ...
+}
+```
+
+## Workflow Interface
+
+The Workflow Interface is a Java interface and is annotated with `@WorkflowInterface`. Workflow interface methods must have one of the following annotations:
+
+- **@WorkflowMethod** denotes the starting point of Workflow Execution. Workflow Execution completes when this methods returns.
+- **@SignalMethod** indicates that this method is a Signal handler method and that it can react to external Signals. It can have parameters which can contain the Signal payload. It does not return a value, so it must have a `void` return type.
```

Comment:
```suggestion
- **@SignalMethod** indicates that this method is a Signal handler method and that it can react to external Signals.
  It can have parameters that can contain the Signal payload.
  It does not return a value, so it must have a `void` return type.
```